### PR TITLE
Create hello

### PR DESCRIPTION
**Pull request by vancuongngo**
new pr

### CODE REVIEW

**Comment by cuongngo-kry** [comment]
hehehe

**Comment by cuongngo-kry** [comment]
hoho

**Comment by cuongngo-kry** [comment]
1

**Comment by cuongngo-kry** [comment]
two two

**Comment by cuongngo-kry** [comment]
123

**Comment by cuongngo-kry** [comment]
456

**Comment by cuongngo-kry** [comment]
789

**Comment by cuongngo-kry** [comment]
1011

File: hello [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hello
+world
```
Comment (cuongngo-kry) [inline hello:2]: 22

**Comment by vancuongngo** [comment]
hej

File: hello (hunk 1 above)
Comment (vancuongngo) [reply hello:2]: 33

**Review by cuongngo-kry — APPROVED** [review]

**Review by cuongngo-kry — CHANGES_REQUESTED** [review]
dddd

**Review by cuongngo-kry — APPROVED** [review]

**Pull request merged**